pandas/core/arrays/datetimes.py

```diff
@@ -322,9 +324,20 @@ def __init__(self, values, dtype=_NS_DTYPE, freq=None, copy=False):
         self._dtype = dtype
         self._freq = freq
 
+        if inferred_freq is None and freq is not None:
+            type(self)._validate_frequency(self, freq)
+
     @classmethod
-    def _simple_new(cls, values, freq=None, dtype=None):
-        return cls(values, freq=freq, dtype=dtype)
+    def _simple_new(cls, values, freq=None, dtype=_NS_DTYPE):
+        assert isinstance(values, np.ndarray)
+        if values.dtype == 'i8':
+            values = values.view(_NS_DTYPE)
+
+        result = object.__new__(cls)
+        result._data = values
+        result._freq = freq
+        result._dtype = dtype
```

Comment:
Is it assumed that the caller is responsible for timezone normalization?

Reply:
yah, assuming at this point we've got valid dtype

Reply:
should the timezones.tz_standardize just go in the Datetime64TZDType constructor?

Reply:
Just to be clear, this may be a non-issue. We may not have anywhere on master where we extract a scalar and build a new DatetimeTZDtype from that scalars tz.

> should the timezones.tz_standardize just go in the Datetime64TZDType constructor?
 
That's a reasonable idea. AFAIK, DatetimeTZDtype is only used on arrays. I'm not sure what the value of a standalone DatetimeTZDtype with a non-normalized tz is.